homeassistant/components/analytics/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""Send instance and usage analytics."""
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.event import async_call_later, async_track_time_interval
+
+from .analytics import Analytics
+from .const import ATTR_HUUID, ATTR_PREFERENCES, DOMAIN, INTERVAL
+
+
+async def async_setup(hass: HomeAssistant, _):
+    """Set up the analytics integration."""
+    huuid = await hass.helpers.instance_id.async_get()
+    analytics = Analytics(hass, huuid)
+
+    # Load stored data
+    await analytics.load()
+
+    # Wait 15 min with first send
+    async_call_later(hass, 900, analytics.send_analytics)
```

Comment:
Should we wait with setting this timer until the Home Assistant started event has occurred?